test/internet/test-dns-promises-resolve.js

```diff
@@ -12,7 +12,7 @@ const dnsPromises = require('dns').promises;
     {
       code: 'ERR_INVALID_ARG_VALUE',
       name: 'TypeError',
-      message: `The argument 'rrtype' is invalid. Received '${rrtype}'`
+      message: `The 'rrtype' is invalid. Received '${rrtype}' argument`
```

Comment:
Is this one correct? Or did argument end up in the wrong place in this string?

Reply:
Oh, good catch. Do we have a make command to run *all* tests?

Reply:
`make test-all-suites`

I would have used `make test-all` but it was already taken.